zoo/pooltool/sum_to_three/config/sum_to_three_vector_obs_sac_config.py

```diff
@@ -0,0 +1,55 @@
+from easydict import EasyDict
```

Comment:
Hello, thank you for your patience, I noticed that in our previous settings, the `model_update_ratio` was using the default value of 0.1, which resulted in too few updates given the same number of environment steps. Additionally, since `replay_buffer_size` is set to 1e6 and `reanalyze_ratio` is 0, many of the target strategies in the buffer are close to random, leading to poor performance. After adjusting the `model_update_ratio` from 0.1 to 1, the current experimental results are as follows (latest commit: https://github.com/opendilab/LightZero/pull/227/commits/5bdcf48607eb6c0f1c056ea5ffd9e9016294a71b):

For `image-obs` (reanalyze_ratio = 0.):
- Data collection
![image](https://github.com/opendilab/LightZero/assets/48008469/032f7772-5410-4c3a-9cce-e9609d8785fa)

- Data evaluation
![image](https://github.com/opendilab/LightZero/assets/48008469/e30137ae-2ddd-4f96-accd-98e95a147cd4)


For `vector-obs` (reanalyze_ratio = 0.25):
- Data collection
![image](https://github.com/opendilab/LightZero/assets/48008469/e634caa0-3a34-46d2-ac97-f210127d0655)

- Data evaluation
![image](https://github.com/opendilab/LightZero/assets/48008469/9880fc47-1a29-4922-8578-da895a80bfdd)


It can be observed that `image-obs` (reanalyze_ratio = 0.) is already able to achieve performance similar to SAC, but the performance of `vector-obs` is still similar to your previous run results. We speculate that this might be because the 4-dimensional vector in `vector-obs` provides less information compared to the (5, 100, 50) setting in `image-obs`. Therefore, my question is whether the 4-dimensional vector in `vector-obs` already provides all the information necessary for optimal decision-making? Currently, experiments with the `image-obs` setting (reanalyze_ratio = 0.25 and variants with reduced `replay_buffer_size`) are still ongoing, and we will update the results once the experiments are completed.

Reply:
Hello, I have supplemented some experimental results of image-obs (`reanalyze_ratio`=0.25) here. We conducted three variant experiments: `condition-sigma rbs1e6`, `condition-sigma rbs1e5`, and `fixed-sigma rbs1e6`. Specifically, these experiments handled the variance of the policy in different ways, namely sigma dependent on input learning (referred to as `condition-sigma`) and fixed sigma=0.3 (referred to as `fixed-sigma`). Additionally, the size of the replay buffer was adjusted, denoted as `rbs-x`. The results show that the performance of these variants is consistent with expectations and significantly better than the results with `reanalyze_ratio=0`.
- collect
![image](https://github.com/opendilab/LightZero/assets/48008469/e95d2837-a079-40cd-b051-d9e3b333c573)

- eval
![image](https://github.com/opendilab/LightZero/assets/48008469/2b3d3f00-8280-4cbe-afe5-d0eb112dc7f8)